bin/publish-modules

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env node
+
+var exec = require( 'child-process-promise' ).exec;
+var semver = require( 'semver' );
+
+console.log( 'Modules requiring publish:' );
+
+function getAllModules() {
+	var command = 'find client -type f -print | grep \'package.json\'';
```

Comment:
Resolved in: https://github.com/belcherj/wp-calypso/commit/858f9e4f4457aae40ddb6917e5513d4b54fb1d0a